definitions/private-law/json/AuthorisationCaseField/ConfidentialTab/AuthorisationCaseField.json

```diff
@@ -26,5 +26,33 @@
     "CaseFieldID": "childrenConfidentialDetails",
     "UserRole": "caseworker-privatelaw-courtadmin",
     "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "02/12/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "applicantsConfidentialDetails",
+    "UserRole": "caseworker-privatelaw-judge",
+    "CRUD": "CRUD"
```

Comment:
why do we have CRUD? Readonly is not enough?

Reply:
For court admin we kept CRUD, so to keep same access level for gatekeeping staff we placed it.